torchgen/model.py

```diff
@@ -1185,6 +1403,43 @@ def strip_ret_annotation(r: Return) -> Return:
         if strip_view_copy_name and base_name.endswith("_copy"):
             base_name = base_name.replace("_copy", "")
 
+        # find mutable inputs that are not originally returned, and convert them to returns
```

Comment:
for future readers, it would be helpful to see a before and after schema example in the comments here